package.json

```diff
@@ -23,14 +21,17 @@
     "mocha": "^2.3.4",
     "strip-ansi": "^3.0.0"
   },
+  "license": "ISC",
+  "main": "./bin/yamscripts.js",
```

Comment:
This was pointing to a non-existent "index.js" previously. Not sure if this should point to "./bin/" or "./lib/", but figured I had a 50/50 chance.